1.0.0-beta7-coreclr/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:jessie
+
+ENV DNX_VERSION 1.0.0-beta7
+ENV DNX_USER_HOME /opt/dnx
+
+RUN apt-get -qq update && apt-get -qqy install unzip curl libunwind8 gettext libssl-dev libcurl3-dev zlib1g
+
+RUN curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_USER_HOME=$DNX_USER_HOME DNX_BRANCH=v$DNX_VERSION sh
+RUN bash -c "source $DNX_USER_HOME/dnvm/dnvm.sh \
+	&& dnvm upgrade -r coreclr"
```

Comment:
I think it is better do it like the other versions. This implicitly downloads the latest version of the runtime and doesn't set the alias which is required for the PATH.

```bash
RUN bash -c "source $DNX_USER_HOME/dnvm/dnvm.sh \
    && dnvm install $DNX_VERSION -r coreclr -a default \
    && dnvm alias default | xargs -i ln -s $DNX_USER_HOME/runtimes/{}$DNX_USER_HOME/runtimes/default"
```